### ISSUE TITLE
moz-spdy-indicator doesn't work with e10s

### ISSUE THREAD

**Issue by ck-on** [chengsun/moz-spdy-indicator#13]
First version of firefox to have a problem

```
Error: NS_ERROR_UNEXPECTED: Component returned failure code: 0x8000ffff (NS_ERROR_UNEXPECTED) [nsILoadContext.associatedWindow]
Source File: chrome://spdyindicator/content/indicator.jsm
Line: 25
```

**Comment by heavensrevenge** [comment]
https://github.com/gomita/firegestures/issues/91 FireGestures had to fix this in their https://github.com/gomita/firegestures/tree/e10s , maybe looking at how they fixed it may help you :)

**Comment by chengsun** [comment]
Should be fixed with version 2.3 (pending review on AMO). Report back if you still have issues.
